src/main/resources/META-INF/plugin.xml

```diff
@@ -205,6 +215,8 @@
         <importer implementation="com.demonwav.mcdev.platform.bukkit.maven.PaperMavenImporter"/>
         <importer implementation="com.demonwav.mcdev.platform.bukkit.maven.PaperMavenImporter"/>
         <importer implementation="com.demonwav.mcdev.platform.bungeecord.maven.BungeeCordMavenImporter"/>
+        <importer implementation="com.demonwav.mcdev.platform.canary.maven.CanaryMavenImporter"/>
+        <importer implementation="com.demonwav.mcdev.platform.canary.maven.NeptuneMavenImporter"/>
```

Comment:
Both to under `<importer implementation="com.demonwav.mcdev.platform.sponge.maven.SpongeMavenImporter"/>`.